system-tests/src/test/java/org/eclipse/edc/samples/transfer/streaming/Streaming02KafkaToHttpTest.java

```diff
@@ -108,7 +108,7 @@ void streamData() {
 
         var destination = Json.createObjectBuilder()
                 .add("type", "HttpData")
-                .add("baseUrl", "http://localhost:" + httpReceiverPort)
+                .add("baseUrl", "http://localhost:4000")
                 .build();
 
         var transferProcessId = CONSUMER.requestAssetFrom("kafka-stream-asset", PROVIDER)
```

Comment:
To make this test cover the sample, this call should be changed in using the json payload in the resource folders as done in [sample streaming-01](https://github.com/eclipse-edc/Samples/pull/335)

Reply:
Fixed. I used all the json payload in the sample this time